docs/src/pages/components/timeline/timeline.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Timeline React component
+components: Timeline, TimelineItem, TimelineSeparator, TimelineDot, TimelineConnector, TimelineContent, TimelineOppositeContent
+---
+
+# Timeline
+
+<p class="description">The timeline displays a list of events in chronological order.</p>
+
+**Note:** This component is not documented in the [Material Design guidelines](https://material.io/), but Material-UI supports it.
+
+## Simple timeline
+
+Simeple timeline showing list of events.
+
+{{"demo": "pages/components/timeline/SimpleTimeline.js"}}
+
+## Right aligned timeline
+
+Right aligned timeline can show the timeline on the right side of the events.
```

Comment:
Beginning with “right aligned timeline“ seems redundant when it goes on to describe a right aligned timeline?

Reply:
Good point :D

Reply:
I've changed a bit the description, please check